test/legacy_test/CMakeLists.txt

```
@@ -1348,6 +1348,11 @@ if(NOT WITH_GPU)
   list(REMOVE_ITEM PIR_COVERAGE_TESTS test_fused_feedforward_pass)
 endif()
 
+set(PIR_COVERAGE_TESTS test_fuse_elewise_add_act_pass)
```

这样会把上面的覆盖掉，下面这样把PIR_COVERAGE_TESTS变成列表吧，不然你就把另外一个单测删了
```suggestion
set(PIR_COVERAGE_TESTS 
test_fused_feedforward_pass 
test_fuse_elewise_add_act_pass)
```